frida-gum/src/instruction_writer.rs

```diff
@@ -438,15 +438,15 @@ impl InstructionWriter for X86InstructionWriter {
         }
     }
 
-    fn put_bytes(&self, bytes: &[u8]) {
-        println!("put_bytes: {:?}", bytes);
+    fn put_bytes(&self, bytes: &[u8]) -> bool {
         unsafe {
-            gum_sys::gum_x86_writer_put_bytes(self.writer, bytes.as_ptr(), bytes.len() as u32)
+            gum_sys::gum_x86_writer_put_bytes(self.writer, bytes.as_ptr(), bytes.len() as u32);
+            true
```

Comment:
Interesting that x86 does not return a boolean but all other architectures do...